dev-tools/integration/.env

```diff
@@ -1,4 +1,4 @@
-ELASTICSEARCH_VERSION=8.12.0-9bbde39d-SNAPSHOT
+ELASTICSEARCH_VERSION=8.13.0-SNAPSHOT
```

Comment:
we don't want this changed

Reply:
I tried without this change, but got an error: https://buildkite.com/elastic/fleet-server/builds/2707#018c5e3a-123b-4f1d-9291-8554076c9e16
Do you know how to fix this for 8.12 branch?
```

ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref f0f8a234-0b83-4322-8d64-75e1f2e26c9c::udwimk7azaiqzaaga61ic5ouq: failed to walk /var/lib/docker/tmp/buildkit-mount1900254152/cover/fleet-server-8.12.0-SNAPSHOT-linux-x86_64: lstat /var/lib/docker/tmp/buildkit-mount1900254152/cover/fleet-server-8.12.0-SNAPSHOT-linux-x86_64: no such file or directory
--
  | make: *** [Makefile:348: build-e2e-agent-image] Error 1
  | 🚨 Error: The command exited with status 2


```